parts/linux/cloud-init/artifacts/health-monitor.sh

```diff
@@ -6,14 +6,27 @@
 set -o nounset
 set -o pipefail
 
+# node could very well be running both dockerd and containerd. we just care about the runtime that kubelet is using.
+getKubeletRuntime() {
+  kubeletArgs=$(systemctl show kubelet | grep 'ExecStart=')
+  runtimeArg=$(echo ${kubeletArgs##*--container-runtime-endpoint=} | cut -f1 -d" ")
+  if [[ "${runtimeArg}" == *containerd.sock ]]; then
+    echo "containerd"
+  else 
+    echo "docker"
```

Comment:
if you switch this to dockerd, you can use `container_runtime_name` directly on L43

Reply:
ah, I see on L48 systemctl already works with docker (no d)? so maybe I'm mistaken

Reply:
yea the systemd unit is `docker` but the actual process is called `dockerd` :(